utils/str.ts

```diff
@@ -77,6 +77,14 @@ export const conversationName = (
 export const formatGroupName = (topic: string, groupName?: string) =>
   groupName || capitalize(humanize(topic.slice(14, 46), 3, " "));
 
+export const formatEphemeralUsername = (address: string, username?: string) =>
+  username || humanize(address.slice(2, 42), 2, "");
+
+export const formatEphemeralDisplayName = (
+  address: string,
+  displayName?: string
+) => displayName || capitalize(humanize(address.slice(2, 42), 2, "", true));
```

Comment:
Why is capitalize called here and then also in humanize?

Reply:
Good catch, does not need to be called since the first word will already be capitalized.